service/routes.py

```diff
@@ -76,3 +74,45 @@ def read_order(order_id):
         )
 
     return jsonify(order.serialize()), status.HTTP_200_OK
+
+######################################################################
+# LIST ALL ORDERS
+######################################################################
+@app.route("/orders", methods=["GET"])
+def list_accounts():
```

Comment:
Func name should be list_orders

Reply:
Oh god. How did this get in there! My bad, fixing it.